src/languages/params.ts

```diff
@@ -417,6 +417,10 @@ type BillingBannerCardOnDisputeParams = {amountOwed: string; cardEnding: string}
 
 type TrialStartedTitleParams = {numOfDays: number};
 
+type EarlyDiscountTitleParams = {discountType: number};
```

Comment:
```suggestion
type EarlyDiscountTitleParams = {discountType: number};
```

Shouldn't this be called `discountAmount`?